pkg/detector/ospkg/suse/suse.go

```diff
@@ -38,10 +38,11 @@ var (
 		"12.5": time.Date(2024, 10, 31, 23, 59, 59, 0, time.UTC),
 		"15":   time.Date(2019, 12, 31, 23, 59, 59, 0, time.UTC),
 		"15.1": time.Date(2021, 1, 31, 23, 59, 59, 0, time.UTC),
-		// 6 months after SLES 15 SP3 release
-		"15.2": time.Date(2021, 10, 31, 23, 59, 59, 0, time.UTC),
+		"15.2": time.Date(2021, 12, 31, 23, 59, 59, 0, time.UTC),
 		// 6 months after SLES 15 SP4 release
-		// "15.3":   time.Date(2028, 7, 31, 23, 59, 59, 0, time.UTC),
+		"15.3":   time.Date(2028, 7, 31, 23, 59, 59, 0, time.UTC),
```

Comment:
Thank you for Pull Request. 

Is this information confirmed?
I think that this is unnecessary until the release of SLE SP4 is confirmed.

Reply:
this is the end of the codestream, so of all the 15.x releases. this date is confirmed. the end date of 15.3 at this point in time is not known yet. 

without that patch, you do get this warning:

```
2021-06-10T19:25:28.585+0200    WARN    This OS version is not on the EOL list: suse linux enterprise server 15.3
```

and thats the key that I'm fixing, in addition to correcting the 15.2 end date.